### PR TITLE
add temurin 19 release blog

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

**Review by smlambert — APPROVED** [review]
LGTM
Is it worth noting that the vthreads preview feature is only available on x64 and aarch64 architectures?

**Comment by gdams** [comment]
paging @sxa incase there are any infra announcments

File: content/blog/eclipse-temurin-19-available/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* [JEP 427: Pattern Matching for switch (Third Preview)](https://openjdk.java.net/jeps/427)
+
+### Library Tools
+
+* [JEP 424: Foreign Function & Memory API (Preview)](https://openjdk.java.net/jeps/424).
+* [JEP 426: Vector API (Fourth Incubator)](https://openjdk.java.net/jeps/426).
+
+### Project Loom Preview/Incubator Features
+
+* [JEP 425: Virtual Threads (Preview)](https://openjdk.java.net/jeps/425) (x86_64 and aarch64 only.
```
Comment (smlambert) [inline content/blog/eclipse-temurin-19-available/index.md:30]:
```suggestion
* [JEP 425: Virtual Threads (Preview)](https://openjdk.java.net/jeps/425) (x86_64 and aarch64 only).
```

**Review by karianna — APPROVED** [review]

**Comment by sxa** [comment]
> paging @sxa incase there are any infra announcments

Nothing special from the infra side in this release to my knowledge. But bear in mind we still have two platforms left to ship at the time of writing.

**Pull request merged**